theano/sandbox/rng_mrg.py

```diff
@@ -940,16 +969,16 @@ def multinomial(self, size=None, n=1, pvals=None, ndim=None, dtype='int64',
                 "which does not use the ndim argument.")
         if pvals.ndim == 2:
             size = pvals[:, 0].shape * n
-            unis = self.uniform(size=size, ndim=1, nstreams=nstreams, **kwargs)
+            unis = self.uniform(size=size, ndim=1, **kwargs)
             op = multinomial.MultinomialFromUniform(dtype)
             n_samples = as_tensor_variable(n)
             return op(pvals, unis, n_samples)
         else:
             raise NotImplementedError(("MRG_RandomStreams.multinomial only"
                                        " implemented for pvals.ndim = 2"))
 
-    def choice(self, size=1, a=None, replace=True, p=None, ndim=None,
```

Comment:
You changed many thing unrelated to your PR. Can you remove those commits instead of reverting all the changes? Hre, nstreams must stay and the order of parameter must not change. Those 2 changes change the user interface.

Reply:
@nouiz I'm sorry

Does this mean I should also delete the binomial implementation for `n>1`?

`nstreams` is still there in some sense. It will just be passed through `kwargs`. I personally find it cleaner to use `kwargs` to simply pass keyword arguments to other functions. It also allows for easier extensibility (just add another `kwarg` to uniform and it will work). 

This being said, I will just do as you ask.